modules/lws-config.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+// * ai_workloads/leader_worker_set/lws-managing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lws-config_{context}"]
+= Deploying a leader worker set
+
+You can use the {lws-operator} to deploy a leader worker set to assist with managing distributed workloads across nodes.
+
+.Prerequisites
+
+* You have installed the {lws-operator}.
+
+.Procedure
+
+. Create a new project by running the following command:
++
+[source,terminal]
+----
+$ oc new-project my-namespace
+----
+
+. Create a file named `leader-worker-set.yaml`
++
+[source,yaml]
+----
+apiVersion: leaderworkerset.x-k8s.io/v1
+kind: LeaderWorkerSet
+metadata:
+  generation: 1
+  name: my-lws <1>
+  namespace: my-namespace <2>
+spec:
+  leaderWorkerTemplate:
+    leaderTemplate: <3>
+      metadata: {}
+      spec:
+        containers:
+        - image: nginxinc/nginx-unprivileged:1.27
+          name: leader
+          resources: {}
+    restartPolicy: RecreateGroupOnPodRestart <4>
+    size: 3 <5>
+    workerTemplate: <6>
+      metadata: {}
+      spec:
+        containers:
+        - image: nginxinc/nginx-unprivileged:1.27
+          name: worker
+          ports:
+          - containerPort: 8080
+            protocol: TCP
+          resources: {}
+  networkConfig:
+    subdomainPolicy: Shared <7>
+  replicas: 2 <8>
+  rolloutStrategy:
+    rollingUpdateConfiguration:
+      maxSurge: 1 <9>
+      maxUnavailable: 1
+    type: RollingUpdate
+  startupPolicy: LeaderCreated
+----
+<1> Specify the name of the leader worker set resource.
+<2> Specify the namespace for the leader worker set to run in.
+<3> Specify the pod template for the leader pods.
+<4> Specify the restart policy for when pod failures occur. Allowed values are `RecreateGroupOnPodRestart` to restart the whole group or `None` to not restart the group.
+<5> Specify the number of pods to create for each group, including the leader pod. For example, a value of `3` will create 1 leader pod and 2 worker pods. The default value is `1`.
```

Comment:
```suggestion
<5> Specify the number of pods to create for each group, including the leader pod. For example, a value of `3` creates 1 leader pod and 2 worker pods. The default value is `1`.
```
Just a nit to avoid future tense.